draft-ietf-taps-interface.md

```diff
@@ -1606,26 +1606,37 @@ Preconnection -> EstablishmentError<reason?>
 
 ## Connection Groups {#groups}
 
-Entangled Connections can be created using the Clone Action:
+Connection Groups can be created using the Clone Action:
 
 ~~~
-Connection := Connection.Clone()
+Connection := Connection.Clone(framer?)
 ~~~
 
-Calling Clone on a Connection yields a group of Connections: the parent
+Calling Clone on a Connection yields a Connection Group containing two Connections: the parent
 Connection on which Clone was called, and a resulting cloned Connection.
 The new Connection is actively openend, and it will send a Ready Event or an EstablishmentError Event.
-The Connections within a group are "entangled" with each other, and become part of a Connection
-Group. Calling Clone on any of these Connections adds another Connection to
-the Connection Group, and so on. "Entangled" Connections share all
-Connection Properties except `Connection Priority` (see {{conn-priority}}) .
-Like all other Properties, Connection Priority is copied 
-to the new Connection when calling Clone(), but it is not entangled: Changing 
-Connection Priority on one Connection does not change it on the other Connections 
-in the same Connection Group. 
-
-The stack of Message Framers associated with a Connection are also copied to 
-the cloned Connection when calling Clone. In other words, a cloned Connection 
+Calling Clone on any of these Connections adds another Connection to
+the Connection Group, and so on. Connections in a Connection Group share all
+Connection Properties except `Connection Priority` (see {{conn-priority}}),
+and these Connection Properties are entangled: Changing one of the
+Connection Properties on one Connection in the Connection Group
+automatically changes it for all others. For example, changing
+`Timeout for aborting Connection` (see
+{{conn-timeout}}) on one Connection in a Connection Group will automatically
+change this Connection Property for all other Connections in the Connection Group in the same way.
```

Comment:
Suggest: 
/will automatically make the same change to this Connection Property for all other Connections in the Connection Group./